smudge/smudge.go

```diff
@@ -20,9 +20,47 @@ import (
 	"flag"
 	"fmt"
 	"github.com/clockworksoul/smudge"
-	"log"
+	"github.com/rs/zerolog/log"
```

Comment:
Adding this dependency breaks `make image` and the build instructions in the README. Would you mind adding the `go get` to the Dockerfile and update the README appropriately?

Reply:
You are right. I was not intended to commit `smudge.go`. It was my PoC test for adding an external logger. I will remove it from the PR.